docs/data/material/experimental-api/classname-generator/classname-generator.md

```diff
@@ -124,16 +124,36 @@ Always create an initializer file to hoist the `ClassNameGenerator` call to the
 
 ```js
 // create a new file called `MuiClassNameSetup.js` at the root or src folder.
+'use client';
```

Comment:
This is not needed in non-RSC frameworks, but I'm not sure if adding a comment about it is worth it.